msbuild/Xamarin.MacDev.Tasks.Core/Tasks/DetectSigningIdentityTaskBase.cs

```diff
@@ -48,7 +48,6 @@ public abstract class DetectSigningIdentityTaskBase : Task
 		[Required]
 		public string SdkPlatform { get; set; }
 
-		[Required]
```

Comment:
This is why the Mac targets had to set SdkIsSimulator="false" in the targets file